src/BoundaryConditions/field_boundary_conditions.jl

```diff
@@ -66,7 +75,7 @@ function FieldBoundaryConditions(indices::Tuple, west, east, south, north, botto
 end
 
 FieldBoundaryConditions(indices::Tuple, bcs::FieldBoundaryConditions) =
-    FieldBoundaryConditions(indices, (getproperty(bcs, side) for side in propertynames(bcs))...)
+    FieldBoundaryConditions(indices, (getproperty(bcs, side) for side in propertynames(bcs)[1:7])...)
```

Comment:
what's 1:7 here? does it deserve a comment?

Reply:
Looking at `FieldBoundaryConditions`, it's the first 7 boundaries: west, east, south, north, bottom, top, immersed.

I think a better way to write this code would be to define a constant, eg

```julia
const boundarynames = (:west, :east, :south, :north, :bottom, :top, :immersed)
```